pkg/services/accesscontrol/database/resource_permissions_test.go

```diff
@@ -16,31 +16,34 @@ import (
 )
 
 type setUserResourcePermissionTest struct {
-	desc       string
-	orgID      int64
-	userID     int64
-	actions    []string
-	resource   string
-	resourceID string
-	seeds      []types.SetResourcePermissionCommand
+	desc              string
+	orgID             int64
+	userID            int64
+	actions           []string
+	resource          string
+	resourceID        string
```

Comment:
nit: similarly, in test structs here and below `resourceID` might not necessarily be ID, right?